### PR TITLE
fixed small leaks from Fixture

### PR DESCRIPTION
**Pull request by sebastianbaginski**
@marcusnaslund

### CODE REVIEW

File: source/unit/Fixture.ooc [hunk 1]
```diff
@@ -122,7 +122,9 @@ Fixture: abstract class {
 	is ::= static IsConstraints new()
 	testsFailed: static Bool { get { This _testsFailed } }
 	printFailures: static func {
-		"Total time: %.2f s" printfln(This totalTime)
+		timeString := "Total time: %.2f s" format(This totalTime)
+		This _print(timeString)
```
Comment (marcusnaslund) [inline source/unit/Fixture.ooc]: If `_print` would free its incoming argument we could get rid of a lot of `free()` calls to strings in this file.

File: source/unit/Fixture.ooc [hunk 2]
```diff
@@ -122,7 +122,9 @@ Fixture: abstract class {
...
+		timeString free()
 		if (This failureNames && (This failureNames count > 0)) {
 			"Failed tests: %i [" printf(This failureNames count)
```
Comment (marcusnaslund) [inline source/unit/Fixture.ooc]: This should use `_print` in the same way I guess

File: source/unit/Fixture.ooc (hunk 1 above)
Comment (sebastianbaginski) [inline source/unit/Fixture.ooc]: That's true, I will update it.

**Comment by sebastianbaginski** [comment]
Updated.

File: source/unit/Fixture.ooc [hunk 3]
```diff
@@ -122,9 +120,9 @@ Fixture: abstract class {
 	is ::= static IsConstraints new()
 	testsFailed: static Bool { get { This _testsFailed } }
 	printFailures: static func {
-		"Total time: %.2f s" printfln(This totalTime)
+		This _print("Total time: %.2f s" format(This totalTime))
```
Comment (marcusnaslund) [inline source/unit/Fixture.ooc]: You're missing a newline here now.

Comment (marcusnaslund) [inline source/unit/Fixture.ooc]: `Total time: 36.76 sFailed tests: 3 [FloatVector3D, Quaternion, FloatVector2D]`

Comment (sebastianbaginski) [inline source/unit/Fixture.ooc]: Thanks, fixed

**Pull request merged**